lib/index.js

```diff
@@ -100,17 +100,33 @@ function validate (opts = {}) {
  * @private
  */
 function transform (post) {
+  const mapItem = (item) => {
+    item.advanced = item.advanced || item.content.advanced
+    item.content = item.content || {}
+    if (item.advanced) {
+      item.advanced.map((c) => {
+        c.fields.map((field) => {
+          if (field.class === "repeater") {
+            item.content[field.name] = field.fields.map((row) => row[0])
+          } else if (field.class === "relationship" && field.value) {
+            item.content[field.name] = field.value
+            item.content[field.name].map(mapItem)
+          } else {
+            item.content[field.name] = field.value
+          }
+        })
+      })
+      delete item.advanced
+    }
+  }
```

Comment:
So it can be called recursively on line 113 to transform the JSON for nested relationship items:
`item.content[field.name].map(mapItem)`